packages/@react-aria/overlays/src/ariaHideOutside.ts

```diff
@@ -85,7 +85,7 @@ export function ariaHideOutside(targets: HTMLElement[], root = document.body) {
 
       // If the parent element of the added nodes is not within one of the targets,
       // and not already inside a hidden node, hide all of the new children.
-      if (![...visibleNodes, ...hiddenNodes].some(node => node.contains(change.target))) {
+      if (![...visibleNodes, ...hiddenNodes].filter(node => !!node).some(node => node.contains(change.target))) {
```

Comment:
This line + the change to the Popover `isOpen` logic in ComboBox fix the intermittent item sizing issue when ComboBox is `defaultOpen/isOpen = true`. Fix suggested by @razvanip, looks reasonable to me, open to other opinions.

Reply:
hmm why would there be null nodes in there? where are they added?

Reply:
i guess it must be due tot the `cond` rendering

checked a little the code, i think it's due to this effect in `useCombobox` 
```
ariaHideOutside([inputRef.current, popoverRef.current]
```

since we use it internally in `ComboBox` and we don't render the `PopOver` until we get the `width` calculated the ref for the `popover` will be null, and will be passed in `ariaHideOutside` 

i can have a deeper look if we need to.

Reply:
i still think it's best not to render the `popover` until we have the `width` calculated, the first render when applying `styling` on the `popover` container  `width` `height` is `null`

Reply:
@razvanip We'll revisit via https://github.com/adobe/react-spectrum/issues/1485, hoping that the fix within the drag and drop pull will fix it.